src/components/Album.jsx

```diff
@@ -0,0 +1,26 @@
+import { Header } from './Header.jsx'
+import { ArtistName } from './ArtistName.jsx'
+import { CoverImage } from './CoverImage.jsx'
+import { AlbumName } from './AlbumName.jsx'
+
+export const Album = ({ dataProp }) => {
+    const albumData = dataProp.albums.items
+    return (
+
+        <div className="album">
+            <Header />
+            {albumData.map((album) => {
+                return (
+
+                    <div className="album-wrapper" key={album.id}>
+
+                        <ArtistName artistNameProp={album} />
```

Comment:
You are almost there Breanna, if you complete the project, pass the different keys from the API to `{album}` with dot notation just like you did with the `key ={album.id}`